### PR TITLE
Update create-integration.md

### PR DESCRIPTION
**Pull request by sakisplus**
Add necessary space between words in line 297

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

- ...

## Links to Magento source code

- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=5062) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=5062) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@sakisplus thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.3/get-started/create-integration.md [hunk 1]
```diff
@@ -294,7 +294,7 @@ Before you can activate your integration in Magento, you must create two pages o
 
 ### Login page {#login}
 
-When a merchant clicks the **Activate** button in Admin, a pop-up login page for the third-party application displays. Magento sends values for `oauth_consumer_key` and `success_call_back` parameters. The application must store the value for`oauth_consumer_key` tie it to the login ID. Use the `success_call_back` parameter to return control back to Magento.
+When a merchant clicks the **Activate** button in Admin, a pop-up login page for the third-party application displays. Magento sends values for `oauth_consumer_key` and `success_call_back` parameters. The application must store the value for `oauth_consumer_key` tie it to the login ID. Use the `success_call_back` parameter to return control back to Magento.
```
Comment (keharper) [inline guides/v2.3/get-started/create-integration.md]:
```suggestion
When a merchant clicks the **Activate** button in Admin, a pop-up login page for the third-party application displays. Magento sends values for `oauth_consumer_key` and `success_call_back` parameters. The application must store the value for `oauth_consumer_key` to tie it to the login ID. Use the `success_call_back` parameter to return control back to Magento.
```

**Pull request merged**